### PR TITLE
UI testing for test generation tab in tool window - 131

### LINKED ISSUE #131: UI testing for Test generation
# Description
Toolwindow test generation has no UI tests for now.

# Definition of Done
* [x]  Test generation tab is tested

### PR DESCRIPTION
**Pull request by SergeyDatskiv**
# Description of changes made
Added a new test file to test the UI for the generated tests tab. The test focus on checking if all the buttons are present in the tab and if the "select all", "deselect all" and "Apply to test suite" buttons work. 

# Why is merge request needed
This pull request is needed to have some UI tests for the generated tests tab in tool window. 

# Other notes
Closes #131

# What is missing?

- [x] I have checked that I am merging into correct branch

### CODE REVIEW

File: src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/GeneratedTestsToolWindowTest.kt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+private const val emptyClass = """public class ArrayUtilsTest {
+}
+"""
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(RemoteRobotExtension::class)
+class GeneratedTestsToolWindowTest {
+
+    private lateinit var editor: EditorFixture
```
Comment (bolkhod) [inline src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/GeneratedTestsToolWindowTest.kt:39]: There are some oddities on my machine (test does not pass), unfortunately

Comment (bolkhod) [reply src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/GeneratedTestsToolWindowTest.kt:39]: Remind from last week: The layout of the buttons has changed from when you initially conceptualized the tests. That's why they fail, most likely.

**Review by kirilvasilev16 — DISMISSED** [review]
I like the way you are creating a new file and applying tests to it

**Review by martinmladenov — APPROVED** [review]
Those tests are a great addition to our extensive test suite and will definitely ensure the quality of our code

**Review by kirilvasilev16 — APPROVED** [review]

**Review by bolkhod — APPROVED** [review]
Now runs perfectly fine!

**Pull request merged**